creator-node/src/routes/files.js

```diff
@@ -15,7 +15,7 @@ const config = require('../config.js')
 const redisClient = new Redis(config.get('redisPort'), config.get('redisHost'))
 const resizeImage = require('../resizeImage')
 const { authMiddleware, syncLockMiddleware, triggerSecondarySyncs } = require('../middlewares')
-const { getIPFSPeerId } = require('../utils')
+const { getIPFSPeerId, ensureMultihashPresent } = require('../utils')
```

Comment:
@hareeshnagaraj should we standardize on CID term?

Reply:
yes, but this is no longer valid as the name has changed